src/api/events/v0/events.ts

```diff
@@ -36,6 +36,14 @@ function newEventServiceClients(): {
   };
 }
 
+export interface PublishOptions {
+  delay?: number;
+}
+
+const DEFAULT_PUBLISH_OPTS: PublishOptions = {
+  delay: 0
+};
+
```

Comment:
Can we add some doc comments here for IDE support to indicate that the delay is in seconds and has min/max values?

Reply:
Done.